### ISSUE TITLE
Missing watchdog in Dockerfile

### ISSUE THREAD

**Issue by jjchiw** [hayer/faas-fsharp-template#3]
I'm not sure if I'm just doing it wrong or I should create a PR, because I think that the bug it's "critical"

When I tried to build the image using

`> faas-cli build -f helloworld.yml`

The output was this:

```
Step 12/22 : COPY --from=watchdog /fwatchdog /usr/bin/fwatchdog
invalid from flag value watchdog: pull access denied for watchdog, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

So I added this line

`FROM openfaas/classic-watchdog:0.18.18 as watchdog`

to the `Dockerfile` https://github.com/hayer/faas-fsharp-template/blob/master/template/fsharp/Dockerfile

and it worked...

**Comment by jjchiw** [comment]
I had other errors with the function sample and some runtime exceptions....

So I create a PR #4

**Comment by hayer** [comment]
Marking as fixed by PR #4.
